.github/workflows/conformance-datapath.yaml

```
@@ -0,0 +1,268 @@
+          done
+
+      - name: Run tests
+        uses: brb/lvh-action@v1.5
+        with:
+          test-name: datapath-conformance
+          image: kind
+          host-mount: ./
+          cmd: |
+            set -eux
+
+            cd /host/
+
+            ./contrib/scripts/kind.sh "" 3 "" "" "none"
+            ./cilium-cli install --wait \
+            --chart-directory=./install/kubernetes/cilium \
+            --helm-set=image.repository=quay.io/${{ github.repository_owner }}/cilium-ci \
+            --helm-set=image.useDigest=false \
+            --helm-set=image.tag=${{ steps.vars.outputs.sha }} \
+            --helm-set=operator.image.repository=quay.io/${{ github.repository_owner }}/operator \
+            --helm-set=operator.image.suffix=-ci \
+            --helm-set=operator.image.tag=${{ steps.vars.outputs.sha }} \
+            --helm-set=operator.image.useDigest=false \
+            --helm-set=hubble.relay.image.repository=quay.io/${{ github.repository_owner }}/hubble-relay-ci \
+            --helm-set=hubble.relay.image.tag=${{ steps.vars.outputs.sha }} \
+            --rollback=false \
+            --config monitor-aggregation=none \
+            --nodes-without-cilium=kind-worker3 --kube-proxy-replacement=strict --helm-set-string="k8sServiceHost=kind-control-plane,k8sServicePort=6443"
+
+            ./cilium-cli connectivity test --datapath
+
+      - name: Fetch artifacts
+        if: ${{ !success() }}
+        uses: brb/lvh-action@v1.5
+        with:
+          provision: 'false'
+          cmd: |
+            cd /host
+            kubectl get pods --all-namespaces -o wide
+            ./cilium-cli status
+            mkdir cilium-sysdumps || true
+            ./cilium-cli sysdump --output-filename cilium-sysdumps/cilium-sysdump-datapath-conformance
+
+      - name: Upload artifacts
+        if: ${{ !success() }}
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8
+        with:
+          name: cilium-sysdumps
+          path: ./cilium-sysdumps
+          retention-days: 5
+
+      - name: Set commit status to success
+        if: ${{ success() }}
+        uses: Sibz/github-status-action@650dd1a882a76dbbbc4576fb5974b8d22f29847f
+        with:
+          authToken: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ steps.vars.outputs.sha }}
+          context: ${{ github.workflow }}
+          description: Datapath tests successful
+          state: success
+          target_url: ${{ env.check_url }}
+
+      - name: Set commit status to failure
+        if: ${{ failure() }}
+        uses: Sibz/github-status-action@650dd1a882a76dbbbc4576fb5974b8d22f29847f
+        with:
+          authToken: ${{ secrets.GITHUB_TOKEN }}
+          sha: ${{ steps.vars.outputs.sha }}
+          context: ${{ github.workflow }}
+          description: Connectivity test failed
```

```suggestion
          description: Datapath test failed
```

just to be consistent with the previous job's description.

---

(nit: also use `tests` for consistency :D)